modules/stripe-next.yml

```diff
@@ -0,0 +1,22 @@
+name: stripe-next
+description: >-
+  This Nuxt module provides an easy way to integrate Stripe in your Nuxt
+  application, both on the client-side and server-side. It utilizes the official
+  stripe package for server-side usage and @stripe/stripe-js for the
+  client-side.
+repo: fuentesloic/nuxt3-stripe
```

Comment:
In general I'd prefer if the module name didn't specifically mention the nuxt version, if possible (https://nuxt.com/docs/guide/going-further/modules#do-not-advertize-with-a-specific-nuxt-version)

otherwise, everything is good for me 👌

Reply:
@danielroe make a lot of sense! fixed with [987c74f](https://github.com/nuxt/modules/pull/677/commits/987c74f2ff9c2108cb5a642e5af44f1ce346471e)